src/services/RedisService.ts

```diff
@@ -0,0 +1,14 @@
+import { createClient, RedisClient } from 'redis';
+import { injectable } from 'inversify';
+
+@injectable()
+export class RedisService {
+    public createNewSession(): RedisClient {
```

Comment:
I suggest we remove word session from method name. This service has nothing to do with session despite session handling being one possible use case. As far as this service is concerned its responsibility is creation of redis client.

Reply:
Changed it to `createClient`